pkg/traceql/combine.go

```diff
@@ -358,6 +358,8 @@ func (q *QueryRangeCombiner) Combine(resp *tempopb.QueryRangeResponse) {
 		q.maxSeriesReached = true
 	}
 
+	// TODO: this is here only for the querier
```

Comment:
long term we need to use the same metric combiner objects everywhere. i don't like that the metrics accumulation logic is still here and is now duplicated in

perhaps something to follow up on?